packages/cactus-plugin-ledger-connector-iroha2/package.json

```diff
@@ -0,0 +1,80 @@
+{
+  "name": "@hyperledger/cactus-plugin-ledger-connector-iroha2",
+  "version": "1.0.0",
+  "description": "Allows Cactus nodes to connect to an Iroha V2 ledger.",
+  "keywords": [
+    "Hyperledger",
+    "Cactus",
+    "Iroha",
+    "Iroha2",
+    "Iroha V2",
+    "Integration",
+    "Blockchain",
+    "Distributed Ledger Technology"
+  ],
+  "homepage": "https://github.com/hyperledger/cactus#readme",
+  "bugs": {
+    "url": "https://github.com/hyperledger/cactus/issues"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/hyperledger/cactus.git"
+  },
+  "license": "Apache-2.0",
+  "author": {
+    "name": "Hyperledger Cactus Contributors",
+    "email": "cactus@lists.hyperledger.org",
+    "url": "https://www.hyperledger.org/use/cactus"
+  },
+  "contributors": [
+    {
+      "name": "Michal Bajer",
+      "email": "michal.bajer@fujitsu.com",
+      "url": "https://www.fujitsu.com/global/"
+    }
+  ],
+  "main": "dist/lib/main/typescript/index.js",
+  "module": "dist/lib/main/typescript/index.js",
+  "types": "dist/types/main/typescript/index.d.ts",
+  "files": [
+    "dist/*"
+  ],
+  "scripts": {
+    "codegen": "run-p 'codegen:*'",
+    "codegen:openapi": "npm run generate-sdk",
+    "generate-sdk": "openapi-generator-cli generate -i ./src/main/json/openapi.json -g typescript-axios -o ./src/main/typescript/generated/openapi/typescript-axios/ --reserved-words-mappings protected=protected"
+  },
+  "dependencies": {
+    "@hyperledger/cactus-common": "1.0.0",
+    "@hyperledger/cactus-core": "1.0.0",
+    "@hyperledger/cactus-core-api": "1.0.0",
+    "@iroha2/client": "3.0.0",
+    "@iroha2/crypto-core": "0.1.1",
+    "@iroha2/crypto-target-node": "0.4.0",
+    "@iroha2/data-model": "3.0.0",
+    "hada": "0.0.8",
+    "rxjs": "7.3.0",
+    "express": "4.17.1",
```

Comment:
@outSH I only saw type imports of express (if this is correct, then) please move express to the `devDependecies` section instead because it's not used at runtime and therefore we don't need to be using resources to install it separately.

Reply:
@petermetz Done

Reply:
@outSH Thank you!